### PR TITLE
Update the note for v12 of Storage SDK

### PR DESCRIPTION
**Pull request by RamjotSingh**

### CODE REVIEW

**Comment by PRMerger16** [comment]
@RamjotSingh : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: data-explorer/kusto/api/netfx/kusto-ingest-client-rest.md [hunk 1]
```diff
@@ -277,8 +277,8 @@ internal static string PrepareIngestionMessage(string db, string table, string d
 Finally, post the message that you constructed, to the selected ingestion queue that you obtained from Azure Data Explorer.
 
 > [!NOTE]
-> .Net storage client, when used, encodes the message to base64 by default. For more information, see [storage docs](https://docs.microsoft.com/dotnet/api/microsoft.azure.storage.queue.cloudqueue.encodemessage?view=azure-dotnet-legacy#Microsoft_WindowsAzure_Storage_Queue_CloudQueue_EncodeMessage).
-If you are NOT using that client, make sure to properly encode the message content.
+> .Net storage client before v12, when used, encodes the message to base64 by default. For more information, see [storage docs](https://docs.microsoft.com/dotnet/api/microsoft.azure.storage.queue.cloudqueue.encodemessage?view=azure-dotnet-legacy#Microsoft_WindowsAzure_Storage_Queue_CloudQueue_EncodeMessage).
```
Comment (vladikbr) [inline data-explorer/kusto/api/netfx/kusto-ingest-client-rest.md:280]:
```suggestion
> .Net storage client versions below v12, when used, encode the message to base64 by default. For more information, see [storage docs](https://docs.microsoft.com/dotnet/api/microsoft.azure.storage.queue.cloudqueue.encodemessage?view=azure-dotnet-legacy#Microsoft_WindowsAzure_Storage_Queue_CloudQueue_EncodeMessage).
```

**Comment by PRMerger20** [comment]
@RamjotSingh : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by vladikbr — APPROVED** [review]

**Comment by RamjotSingh** [comment]
@vladikbr Who else needs to review this PR and merge it? I don't have merge permissions.

**Comment by orspod** [comment]
@RamjotSingh - I will review the changes and merge the PR. Thanks again for your contribution.

File: data-explorer/kusto/api/netfx/kusto-ingest-client-rest.md [hunk 2]
```diff
@@ -277,8 +277,8 @@ internal static string PrepareIngestionMessage(string db, string table, string d
 Finally, post the message that you constructed, to the selected ingestion queue that you obtained from Azure Data Explorer.
 
 > [!NOTE]
-> .Net storage client, when used, encodes the message to base64 by default. For more information, see [storage docs](https://docs.microsoft.com/dotnet/api/microsoft.azure.storage.queue.cloudqueue.encodemessage?view=azure-dotnet-legacy#Microsoft_WindowsAzure_Storage_Queue_CloudQueue_EncodeMessage).
-If you are NOT using that client, make sure to properly encode the message content.
+> .Net storage client versions below v12, when used, encode the message to base64 by default. For more information, see [storage docs](https://docs.microsoft.com/dotnet/api/microsoft.azure.storage.queue.cloudqueue.encodemessage?view=azure-dotnet-legacy#Microsoft_WindowsAzure_Storage_Queue_CloudQueue_EncodeMessage).
+If you are NOT using that client (e.g. using v12 and above of Storage Client), make sure to properly encode the message content.
```
Comment (orspod) [inline data-explorer/kusto/api/netfx/kusto-ingest-client-rest.md:281]:
```suggestion
> .Net storage client versions below v12, by default, encode the message to base64 For more information, see [storage docs](https://docs.microsoft.com/dotnet/api/microsoft.azure.storage.queue.cloudqueue.encodemessage?view=azure-dotnet-legacy#Microsoft_WindowsAzure_Storage_Queue_CloudQueue_EncodeMessage).
If you are using .Net storage client versions above v12, you must properly encode the message content.
```

**Comment by PRMerger20** [comment]
@orspod : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**